uportal-war/src/main/webapp/media/skins/respondr/common/less/navigation.less

```diff
@@ -549,6 +549,10 @@
                                     left: 3px;
                                     position: relative;
                                     top: 0;
+                                    /* increase clickable size of dropdown toggle */
+                                    padding: 0 5px 5px;
+                                    margin: 0 -5px -5px;
+
```

Comment:
Looks as though changing `width: 17px;` to `width: 26px;` on `.dropdown-toggle.portal-navigation-dropdown` under 769px (only) improves the "touchability" of the toggle.